specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2019-08-01/imds.json

```diff
@@ -77,6 +81,10 @@
               "$ref": "#/definitions/ApplicationResponse"
             }
           },
+          "204": {
+            "description": "No application profiles found.",
+            "schema": {}
```

Comment:
Assuming that you actually don't return anything, the `schema` key should be omitted.

Reply:
Agreed. However, when I tried to leave out the `schema` field, the merge validation checks failed, complaining about the missing schema. I settled on this as the workaround for the time being, but I'm happy to revert my change if the merge validation checks are updated accordingly.

Reply:
Nick's suggestion helped me fix this. I've removed the `schema` key.